src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -1382,6 +1370,105 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
+    Partial Class BoundFieldOrPropertyInitializer
+        Implements IFieldInitializer, IPropertyInitializer
```

Comment:
How about I split BoundFieldOrPropertyInitializer into BoundFieldInitializer and BoundPropertyInitializer, and eliminate BoundFieldEqualsValue and BoundPropertyEqualsValue?